java/src/main/native/src/TableJni.cpp

```diff
@@ -3466,7 +3488,15 @@ JNIEXPORT jobjectArray JNICALL Java_ai_rapids_cudf_Table_contiguousSplitGroups(
       n_result.set(
           i, cudf::jni::contiguous_table_from(env, result[i].data, result[i].table.num_rows()));
     }
-    return n_result.wrapped();
+
+    jobjectArray groups_array = n_result.wrapped();
+
+    if (genUniqKeys) {
```

Comment:
```suggestion
    if (group_by_result_table.) {
```
Not sure the gather could return an invalid table, however better to check the returned value here.

Reply:
Checked the doc of `cudf::gather`, and it will not return an invalid table.
https://github.com/rapidsai/cudf/blob/v22.08.00/cpp/include/cudf/copying.hpp#L83

Verified that the result table has 0 rows if the inputted gather map is empty.